test/scripts/e2e-cp.sh

```diff
@@ -154,7 +154,7 @@ fi
 
 # Only run network segmentation tests if they are explicitly requested
 NETWORK_SEGMENTATION_TESTS="Network Segmentation"
-if [ "${WHAT}" != "${NETWORK_SEGMENTATION_TESTS}" ]; then
+if [ "${WHAT}" != ${NETWORK_SEGMENTATION_TESTS}* ]; then
```

Comment:
I think this is causing network segmentation tests to be run on non-network segmentation lanes

Reply:
yeah, @ricky-rav I think you need double brackets:
```suggestion
if [[ "${WHAT}" != ${NETWORK_SEGMENTATION_TESTS}* ]]; then
```